lib/galaxy/tools/extract_dataset.xml

```diff
@@ -40,6 +40,22 @@
   </outputs>
   <help><![CDATA[
 
+========
+Synopsis
+========
+
+Extracs datasets from a collection based on either position or name.
+
+===========
+Description
+===========
+
+The tool allow extracting datasets based on position (**The first dataset** and **Select by index** options) or name (**Select by element identifier** option). If collection contains multiple elements with the same name, all of them will be extracted and a new collection will be created. Finally
```

Comment:
I initiated this process in part because collection warrant (1) separate section in GTN and (2) a dedicated manuscript. So, yes we need schematics illustrating these.